### ISSUE TITLE
Insufficient color contrast for "I'm not a robot" checkbox and "Enter the words you hear" input field

### ISSUE THREAD

**Issue by BarrierbreakApp** [google/recaptcha#426]
Example URL: https://www.google.com/recaptcha/api2/demo

Background/foreground color combinations with an insufficient contrast ratio are found:
• The #C1C1C1 border of "I'm not a robot" checkbox has a contrast ratio of 1.7:1 with #F9F9F9 outer background.
• The #C1C1C1 border of "I'm not a robot" checkbox has a contrast ratio of 1.8:1 with #FFFFFF inner background.
•The #DFDFDF border of "Enter the words you hear" input field (that becomes available on activating "I'm not a robot" checkbox > "Get an audio challenge" button) has a contrast ratio of 1.3:1 with #FFFFFF  background.

Suggestion:
Ensure that the color contrast for all user interface controls in their default as well as active, hover and focus states is 3:1 with the adjacent content.

![reCaptcha_3a](https://user-images.githubusercontent.com/16647640/107732255-eb0e6380-6d1d-11eb-98b7-3b3ed2314795.png)

![reCaptcha_3b](https://user-images.githubusercontent.com/16647640/107732257-ec3f9080-6d1d-11eb-9d67-47652e9875c6.png)

**Comment by rowan-m** [comment]
These all look like automated issues for the widget unrelated to the PHP client library in this repo.
